dist/strap/dbconf/mysql/transactions.mysql

```diff
@@ -57,3 +57,4 @@ Database  transactions  COLUMN_DEF   "comments=text"
 Database  transactions  COLUMN_DEF   "currency_locale=varchar(32)"
 Database  transactions  COLUMN_DEF   "pay_cert_total=varchar(16)"
 Database  transactions  COLUMN_DEF   "pay_cert_ord_total=varchar(16)"
+Database  transactions  COLUMN_DEF   "tax_sent=enum('1','-1') not null default '1'"
```

Comment:
As we discussed, I would love to see some documentation here of the weird MySQL enum behavior that leads to not having an explicit `'0'` value. Possibly including a link to the [MySQL enum docs](https://dev.mysql.com/doc/refman/5.7/en/enum.html#enum-nulls).

Reply:
Change made and pushed.